src/main/java/Main.java

```diff
@@ -42,7 +46,10 @@ public class Main {
     public static void main(String[] args) throws IOException {
 
         log.info("Start generating application with name: {}", NAME);
-
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Kannst du mir den gefallen tun und auch noch die 4 Zeile einfach an den Anfang vom //////// Read the user code from the old project //////// packen. Dann ist wirklich alles gekapselt voneinander unter den jeweiligen Kommentaren <3

Reply:
geht leider nicht, da ich das unten nochmal braue

Reply:
Zeile 150

Reply:
Du musst es ja nicht mit in die if-Bedingung packen. Ich hab nur darum gebeten es unter den Kommentar zu packen :D